src/sql/src/parsers/create_parser.rs

```diff
@@ -547,6 +572,13 @@ impl<'a> ParserContext<'a> {
             )));
         }
 
+        Ok(name)
+    }
+
+    pub fn parse_column_def(&mut self) -> std::result::Result<ColumnDef, ParserError> {
```

Comment:
Prefer to document the public function. @coderabbitai generates comments for this function.